### DISCUSSION TITLE
Release schedules?

### DISCUSSION THREAD

**Discussion by Jackbennett** [govuk-react/govuk-react#1142] [General]
Hi there, Seems to be code sitting in main that hasn't hit NPM yet i.e. the `licence` prop to the footer. I couldn't find a note on when CI is run to make a release, is that just up to maintainers?

**Comment by penx** [comment]
Yes a release requires some manual input from maintainers. The process is in contributing.md

Would you like me to get this released?

I may have some time this week.

**Reply by Jackbennett** [reply]
Yeh a release would be useful, I can wait if it's on the todo list in not-too-long. Mainly I just wanted to know if it was still getting some love.

If I were to follow that process in a pull request for the maintainers would that help in future?

**Reply by penx** [reply]
@Jackbennett sorry for the delay in getting back, I've been away, I've just released govuk-react@0.10.2
